src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -346,16 +397,16 @@ private void CreateColorTexture()
 
         private void CreateOutputTextures()
         {
-            if (videoOutputTexture == null)
+            if (quadViewOutputTexture == null)

```

Comment:
>quadViewOutputTexture [](start = 16, length = 21)

nit: it seems like this is created regardless of whether or not its used/wheter or not quad compositing is kicked off. It may be negligible, but creating a 4K render texture that isn't used seems like bad practice.

Reply:
It's used if you either "view" the texture in the Display tab or are recording to it. For now I left it in the rendering path for simplicity, but this could be cleaned up to not create or use it unless one of those two features are being used.